user-service/src/main/java/uk/gov/gchq/palisade/service/user/model/UserRequest.java

```diff
@@ -31,11 +31,10 @@
 import java.util.StringJoiner;
 
 /**
- * The UserRequest is the input for user-service to identify the User associated with the given User ID.
- * UserResponse is the output for this service which will include the User identified by the service.
+ * The UserRequest is the input for User Service to identify the User associated with the given User ID.
```

Comment:
reword, doesnt make a lot of sense
something like, the UserService accepts a request in the form of a UserRequest object, and contains x

Reply:
I'm not sure the example is quite right as the JavaDoc is for the UserRequest rather than explaining what the User Service does.
I have re-worded this slightly though.